axelrod/tests/unit/test_rand.py

```diff
@@ -24,7 +24,24 @@ def test_strategy(self):
         """Test that strategy is randomly picked (not affected by history)."""
         self.first_play_test(C, seed=1)
         self.first_play_test(D, seed=2)
-        self.responses_test([C], [C, D, C], [C, C, D], seed=1)
+
+        opponent = axelrod.MockPlayer()
+        outcomes = [(C, C), (D, C), (D, C)]
```

Comment:
This tests the histories above but not the next move, so it's not exactly equivalent. Doesn't matter in this case of course.